flocker/provision/_install.py

```diff
@@ -1250,6 +1280,7 @@ def maybe_disable(unit):
 _flocker_uninstallers = {
     "ubuntu-14.04": _uninstall_flocker_ubuntu1404,
     "centos-7": _uninstall_flocker_centos7,
+    "rhel-7.2": _uninstall_flocker_centos7,
```

Comment:
Confirming that 15.10 support is no longer required, based on in-person discussions. We only have client support on 15.10 for benchmarking work.